app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,70 @@
+class MoviesController < ApplicationController
+  before_action :set_movie, only: %i[show edit update destroy]
+
+  # GET /movies or /movies.json
+  def index
+    @movies = Contexts::Movies::Queries::MovieQueries.new.all
+    @movies
```

Comment:
```suggestion
    @movies
```

I suppose previous line  will return the same, isn't it?